lib/src/code_modifiers/indent_code_modifier.dart

```diff
@@ -5,10 +5,10 @@ import 'package:flutter/widgets.dart';
 import '../code_field/editor_params.dart';
 import 'code_modifier.dart';
 
-class IntendModifier extends CodeModifier {
+class IntentModifier extends CodeModifier {
   final bool handleBrackets;
 
-  const IntendModifier({
+  const IntentModifier({
```

Comment:
The file name is `indent_code_modifier`, while the class is renamed to `IntentModifier`.

Reply:
file named such way in a base repo. @alexeyinkin What do you think? Are we still support compatibility with the base repo?